.github/workflows/pr-plugin-build.yml

```
@@ -0,0 +1,199 @@
+          # Check if we have any changes
+          if [ ! -s "changed_files.txt" ]; then
+            echo "No emhttp files changed"
+            echo "has_changes=false" >> "$GITHUB_OUTPUT"
+          else
+            echo "has_changes=true" >> "$GITHUB_OUTPUT"
+          fi
+      
+      - name: Generate plugin version
+        if: steps.changed-files.outputs.has_changes == 'true'
+        id: version
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:3:28: Double quote to prevent globbing and word splitting

(shellcheck)